rewards/rewards_buttons.lua

```diff
@@ -132,7 +132,7 @@ end
 
 local function decorateRewardButton(button, rewardType, reward)
 	if rewardType == Rewards.REWARD_TYPES.CURRENCY then
-		decorateCurrencyButton(button, reward.index, Storyline_API.getCurrentEvent() == "QUEST_PROGRESS" and "required" or "reward", reward.icon, reward.text, reward.count, reward.currencyID, reward.quality);
+		decorateCurrencyButton(button, reward.index, reward.rewardType, reward.icon, reward.text, reward.count, reward.currencyID, reward.quality);

```

Comment:
I think the `"required"` is still needed for quests where some amount of currency is required to complete the quest, as I think (my memory is getting old) these buttons are also used when showing objectives.

Reply:
I mentioned it was already done in the currency objectives :nowo:
https://github.com/Ellypse/Storyline/blob/3de3e98ce78ed59264beb261989293f9d8e9e9d7/rewards/rewards.lua#L474